rust/windows-client/src-tauri/src/client/debug_commands.rs

```diff
@@ -45,6 +61,44 @@ mod details {
     use super::*;
     use std::sync::Arc;
 
+    pub fn open_deep_link(path: &url::Url) -> Result<()> {
+        let subscriber = tracing_subscriber::FmtSubscriber::builder()
```

Comment:
Is this called essentially at the entrypoint of the GUI process? Is this related to the main file logger for the app?

Reply:
In case it's helpful:

https://github.com/firezone/firezone/blob/main/rust/linux-client/src/main.rs#L12

Reply:
When a web browser navigates to our firezone:// URL, it tells Windows to open it, then Windows will launch our exe with the `open-deep-link` subcommand, as we registered it. So that 2nd instance of our app says "Oh I'm not the real app, I just need to forward this link to the main instance and then I'll exit"

I'm no sure what will happen if 2 instances of the app share the same log files but I assume it could be bad, so I didn't want to create another file_logger here. I can add that as a comment in the code.